### PR TITLE
Swift 4 migration

### PR DESCRIPTION
**Pull request by mateuszszklarek**
This pull request contains all required changes for Swift 4.

### CODE REVIEW

File: Adyen.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1197,21 +1201,27 @@
...
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_RANGE_LOOP_ANALYSIS = YES;
+				CLANG_WARN_STRICT_PROTOTYPES = YES;
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 1.4.0;
+				CURRENT_PROJECT_VERSION = 1.5.0;
```
Comment (joostvandijk) [inline Adyen.xcodeproj/project.pbxproj]: Could you remove the version changes? The versioning is defined by Adyen team following a internal release schedule, and we’ll update the version once this is merged to master.

Comment (mateuszszklarek) [inline Adyen.xcodeproj/project.pbxproj]: 👍 great, I will remove version 1.5.0.

**Comment by mateuszszklarek** [comment]
Great to hear @joostvandijk, thanks for your time.

**Comment by joostvandijk** [comment]
Thanks @mateuszszklarek!

**Pull request merged**